invenio_records_marc21/services/config.py

```diff
@@ -90,9 +92,6 @@ class Marc21RecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
     schema_secret_link = None
     review = None
 
-    # Permission policy
```

Comment:
does this change belongs to the resource_type addition?

Reply:
> does this change belongs to the resource_type addition?

No this line is a duplet and gets overwritten later in the config.